docs/Tutorial-fr.md

```diff
@@ -0,0 +1,108 @@
+
+```python
+pdf.set_font('helvetica', 'B', 16)
+```
+
+On aurait pu spécifier une police en italique avec `I`; soulignée avec `U` ou une police normale avec une chaine de caractères vide. Il est aussi possible de combiner les effets en combinant les caractères. Notez que la taille des caractères est à spécifier en points (pts), pas en millimètres (ou tout autre unité); c'est la seule exception.
+Les autres polices fournies par défaut sont `Times`, `Courier`, `Symbol` et `ZapfDingbats`.
+
+On peut maintenant afficher une cellule avec [cell](fpdf/fpdf.html#fpdf.fpdf.FPDF.cell). Une cellule est une zone rectangulaire, avec ou sans cadre, qui contient du texte. Elle est affichée à la position actuelle du curseur. On spécifie ses dimensions, le texte (centré ou aligné), si'l y a une bordure ou non, ainsi que la position du curseur après avoir affiché la cellule (s'il se déplace à droite, vers le bas ou au début de la ligne suivante). Pour ajouter un cadre, on utilise ceci :
+
+```python
+pdf.cell(40, 10, 'Hello World!', 1)
+```
+
+Pour ajouter une nouvelle cellule avec un texte centré, et déplacer le curseur à la ligne suivante on utilise cela :
+
+```python
+pdf.cell(60, 10, 'Powered by FPDF.', ln=1, align='C')
+```
+
+**Remarque** : le saut de ligne peut aussi être fait avec [ln](fpdf/fpdf.html#fpdf.fpdf.FPDF.ln). Cette méthode permet de spécifier la hauteur du saut.
+
+Enfin, le document est sauvegardé à l'endroit spécifié en utilisant [output](fpdf/fpdf.html#fpdf.fpdf.FPDF.output). Sans aucun paramètre, `output()` retourne le buffer `bytearray` du PDF.
+
+## Tuto 2 - En-tête, bas de page, saut de page et image ##
+
+Voici un exemple contenant deux pages avec un en-tête, un bas de page et un logo :
+
+```python
+{% include "../tutorial/tuto2.py" %}
+```
+
+[PDF généré](https://github.com/PyFPDF/fpdf2/raw/master/tutorial/tuto2.pdf)
+
+Cet exemple utilise les méthodes [header](fpdf/fpdf.html#fpdf.fpdf.FPDF.header) et [footer](fpdf/fpdf.html#fpdf.fpdf.FPDF.footer) pour générer des en-têtes et des bas de page. Elles sont appelées automatiquement. Elles existent déjà dans la classe FPDF mais elles ne font rien, il faut donc étendre la classe et ajouter un override pour ces méthodes.
+
+Le logo est affiché avec la méthode [image](fpdf/fpdf.html#fpdf.fpdf.FPDF.image) en spécifiant la position du coin supérieur gauche et la largeur de l'image. La hauteur est calculée automatiquement pour garder les proportions de l'image.
+
+Pour centrer le numéro de page dans le bas de page, il faut passer la valeur nulle à la place de la largeur de la cellule. Cela fait prendre toute la largeur de la page à la cellule, ce qui permet de centrer le texte. Le numéro de page actuel est obtenu avec la méthode [page_no](fpdf/fpdf.html#fpdf.fpdf.FPDF.page_no); le nombre total de pages est obtenu avec la variable `{nb}` qui prend sa valeur quand le document est fermé (si la méthode [alias_nb_pages](fpdf/fpdf.html#fpdf.fpdf.FPDF.alias_nb_pages) est appelée au préalable).
```

Comment:
"(si la méthode alias_nb_pages est appelée au préalable)" n'est plus vrai avec la version actuelle.

On pourrait par contre indiquer : (la méthode alias_nb_pages permet d'employer une autre `string` comme "ancre")

Reply:
Changed to
`la méthode [alias_nb_pages](fpdf/fpdf.html#fpdf.fpdf.FPDF.alias_nb_pages) permet de définir un autre nom de variable pour cette valeur`